### DISCUSSION TITLE
Scaling to a single instance per service - Single Node Mode (non HA)

### DISCUSSION THREAD

**Discussion by bmsan** [prometheus-operator/kube-prometheus#1690] [Q&A]
Hello,

I'd like to scale down the instances of each service to 1, I'm running minikube on a single node and for this usecase running some of the components in Highly Available mode can take a toll on the system resources.

I've tried after deployment to scale manually (with kubectl scale --replicas) the services but at least on of them seemed configured in Cluster Mode and started complaining it cannot see the other members of the cluster.

I'm just getting started with both Kubernetes and prometheus so I am not sure how it would be best to approach the reconfiguration I have in mind.

**Comment by ArthurSens** [comment]
Hello 👋 

The Prometheus and Alertmanager are deployed using Custom Resource Definition (CRD). If you are getting started in Kubernetes, I recommend searching and learning this subject a bit  🙂. Anyway, Prometheus and Alertmanager can be scaled down if you edit the number of replicas in the Prometheus and Alertmanager Custom Resource (CR).


Docs around the CRs Specification can be seen here:
* [Alertmanager](https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/api.md#alertmanagerspec)
* [Prometheus](https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/api.md#prometheusspec)

**Reply by bmsan** [reply]
Thank you @ArthurSens !
